halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/providers/aws/AwsAccountValidator.java

```diff
@@ -17,17 +17,36 @@
 
 package com.netflix.spinnaker.halyard.config.validate.v1.providers.aws;
 
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.services.ec2.AmazonEC2;
+import com.amazonaws.services.ec2.AmazonEC2ClientBuilder;
 import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
 import com.netflix.spinnaker.halyard.config.model.v1.providers.aws.AwsAccount;
 import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
 import com.netflix.spinnaker.halyard.core.problem.v1.Problem.Severity;
+import com.netflix.spinnaker.halyard.core.tasks.v1.DaemonTaskHandler;
 import org.springframework.stereotype.Component;
 
 @Component
 public class AwsAccountValidator extends Validator<AwsAccount> {
   @Override
   public void validate(ConfigProblemSetBuilder p, AwsAccount n) {
-    // TODO(https://github.com/spinnaker/halyard/issues/116)
-    p.addProblem(Severity.WARNING, "No validators exist for this provider.");
+    DaemonTaskHandler.message(String.format("Validating %s with %s", n.getNodeName(), getClass().getSimpleName()));
+
+    AWSCredentialsProvider credentialsProvider = DefaultAWSCredentialsProviderChain.getInstance();
```

Comment:
Using the same method for initializing the credentials gives you a higher degree of confidence that your config will work once it passes Halyard's checks. If it turns out that it's too complicated we can skip this for now and move on to getting this to work on k8s.